common/lib/protocol-definitions/package.json

```diff
@@ -24,15 +45,15 @@
 		}
 	},
 	"main": "dist/index.js",
-	"module": "lib/index.mjs",
 	"types": "dist/index.d.ts",
 	"scripts": {
-		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
-		"build": "npm run build:compile && concurrently npm:lint npm:build:docs",
-		"build:compile": "concurrently npm:typetests:gen npm:tsc npm:build:esnext",
-		"build:docs": "npm run api-extractor:commonjs && npm run api-extractor:esnext",
-		"build:esnext": "tsc-multi --config ../../build/build-common/tsc-multi.esm.json",
+		"api": "fluid-build . --task api",
+		"api-extractor:commonjs": "api-extractor run --config ./api-extractor-cjs.json",
+		"api-extractor:esnext": "api-extractor run --local",
+		"build": "fluid-build . --task build",
+		"build:compile": "fluid-build . --task compile",
+		"build:docs": "fluid-build . --task api",
+		"build:esnext": "tsc --project ./tsconfig.json",
 		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
 		"check:release-tags": "api-extractor run --local --config ./api-extractor-lint.json",
 		"ci:build": "npm run build:compile",
```

Comment:
`ci:build` only calls `build:compile` but now relies on `api` to generate the rollup files. The lint step will fail without `api` being run. I think `ci:build` should be `fluid-build --task ci:build`.

Reply:
ci:build doesn't need the api files, but linting does. Thus I changed the lint task to use fluid-build, but left ci:build calling compile only. Lint is called independently in CI, so when it's run, it'll do the api-extractor stuff too.

Reply:
I approved, but I don't think that is correct. Build and ci:build should produce everything required to publish package. Linting should not generate a portion of the build. There should be a follow up to correct that.